libs/@blockprotocol/graph/src/types/subgraph/edges.ts

```diff
@@ -2,14 +2,14 @@ import { BaseUri } from "@blockprotocol/type-system/slim";
 
 import { EntityId } from "../entity.js";
 import { Timestamp } from "../temporal-versioning.js";
-import {
-  KnowledgeGraphOutwardEdge,
-  OntologyOutwardEdge,
-} from "./edges/outward-edge.js";
+import { KnowledgeGraphOutwardEdge } from "./edges/variants/knowledge.js";
+import { OntologyOutwardEdge } from "./edges/variants/ontology.js";
 
 export * from "./edges/kind.js";
 export * from "./edges/outward-edge.js";
-export * from "./edges/outward-edge-alias.js";
+export * from "./edges/variants.js";
+
+/** @todo - Re-express these and `Vertices` as `Record`s? */
```

Comment:
I'm unsure why I ended up expressing them like this originally, it seems that they'd be better as `Record`s, and perhaps they'd behave slightly nicer sometimes with narrowing.

I don't want to change it here though because I want to make sure it doesn't break any code in usage in MBD, so will probably look into this as part of the follow-up PR.